src/main/java/com/google/neighborgood/servlets/TaskServlet.java

```diff
@@ -93,8 +96,24 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
       }
       out.append("<div class='task-container'>");
       out.append("<div class='task-header'>");
+
+      // Retrieves task's user to get user's nickname
+      String taskOwner = (String) entity.getProperty("Owner");
+      Query userQuery =
+          new Query("UserInfo")
+              .setFilter(new FilterPredicate("userId", FilterOperator.EQUAL, taskOwner));
+      PreparedQuery userResults = datastore.prepare(userQuery);
+      Entity userEntity = userResults.asSingleEntity();
+
+      if (userEntity == null) {
+        System.err.println("Unable to find the user entity based on the current user id");
```

Comment:
"current" is ambiguous; could you say something about how the task's owner couldn't be found?

Also, is it worth failing the whole request if we can't look up one user?

Reply:
Perhaps it would be better to have a fallback user nickname for when the actual nickname could not be found (eg. "Neighbor"). I'll make both of these changes for this, thanks for bringing this up!

Reply:
sounds good to me